src/main/java/com/scalar/db/storage/dynamo/ResultImpl.java

```diff
@@ -131,22 +130,23 @@ private Value convert(AttributeValue itemValue, String name, String type)
       throws UnsupportedTypeException {
     // When itemValue is NULL, the value will be the default value.
     // It is the same behavior as the datastax C* driver
+    boolean isNull = itemValue == null || (itemValue.nul() != null && itemValue.nul());
```

Comment:
Just a question.
Is there any case where `itemValue.nul()` is null?

Reply:
Yes, `AttributeValue#nul()` returns `Boolean`. It returns null when the itemValue has a non-null value.